<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CClassAttributesEditorUI</class>
 <widget class="QWidget" name="CClassAttributesEditorUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Custom Attributes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Attributes of</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ClassId">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="statusTip">
        <string>Choose element to edit default attributes</string>
       </property>
       <item>
        <property name="text">
         <string>graph</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>node</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>edge</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ColorScheme">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="statusTip">
        <string>Apply predefined color scheme to graph</string>
       </property>
       <property name="text">
        <string>Color scheme</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QtTreePropertyBrowser" name="Editor"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="AddButton">
       <property name="statusTip">
        <string>Add new custom attribute for the selected items</string>
       </property>
       <property name="text">
        <string>Add...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ChangeButton">
       <property name="text">
        <string>Change...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RemoveButton">
       <property name="statusTip">
        <string>Remove the attribute from the selected items</string>
       </property>
       <property name="text">
        <string>Remove...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtTreePropertyBrowser</class>
   <extends>QTreeWidget</extends>
   <header location="global">QtTreePropertyBrowser</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ClassId</tabstop>
  <tabstop>ColorScheme</tabstop>
  <tabstop>Editor</tabstop>
  <tabstop>AddButton</tabstop>
  <tabstop>ChangeButton</tabstop>
  <tabstop>RemoveButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
